flow-typed/types.js

```diff
@@ -1,7 +1,54 @@
 // @flow
 
+export type User = {
+  id: number,
+  fullName: string,
+  initials: string,
+  email: string,
+  avatarThumb: string
+};
+
+export type TeamMember = {
```

Comment:
This file will probably get quite large in the future eh? Can we somehow structure this a bit more? At first, I thought about mapping it directly to `ActiveRecords`, but I see there is different data in them, so that's probably not the best option. But we would probably still want to group these types together (e.g. ones related to teams, etc.).

Can we just namespace them somehow? E.g. `Teams.Member`, `Teams.Team`, `Teams.NewTeam`, ...?

Reply:
This is a good idea! I have to check if will work like this or if we use just a convention Teams$Member, Teams$Team, ...